weekly/2023/week-36.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Week 36 in Packit
+date: 2023-09-11
+authors: flachman
+tags:
+  - 2023-September
+  - 2023
+  - September
+---
+
+## Week 36 (September 5th – September 11th)
+
+- Packit now supports `commit-message` action that can be used to override the default commit message produced by Packit during `propose-downstream` or `pull-from-upstream`. Please pay attention to our documentation with regards to the usage of this action. ([packit#2070](https://github.com/packit/packit/pull/2070))
```

Comment:
```suggestion
- Packit now supports `commit-message` action that can be used to override the default commit message produced by Packit during `propose-downstream` or `pull-from-upstream`. Please pay attention to [our documentation](https://packit.dev/docs/configuration/actions#commit-message-1) with regards to the usage of this action. ([packit#2070](https://github.com/packit/packit/pull/2070))
```

Reply:
```suggestion
- Packit now supports `commit-message` action that can be used to override the default commit message produced by Packit during `propose-downstream` or `pull-from-upstream`. Please pay attention to [our documentation](https://packit.dev/docs/configuration/actions#commit-message) with regards to the usage of this action. ([packit#2070](https://github.com/packit/packit/pull/2070))
```

With the `-1` it seems to skip to the env variables